templates/store/categories/iot.html

```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit{% endblock %}
+
+{% block meta_title %}
+  Snapcraft - Fast and secure publishing for Internet of Things
```

Comment:
Suggestion: "Publish, install and update IoT apps - Snapcraft"

Reply:
Done, thanks